src/pages/UserPage/UserPage.tsx

```diff
@@ -38,7 +38,7 @@ export const UserPage = () => {
         projectUrls:[''],
         bio:'',
         expectedContractType: ContractType.NO_PREFERENCE,
-        expectedTypeWork: expectedTypeWorkEntity.IRRELEVANT,
+        expectedTypeWork: expectedTypeWorkEntity.IRRELAVANT,
```

Comment:
Proszę zwracać komentarz jak chcesz coś poprawić a nie dorzucać commit do nie swojego taska.

Nie istnieje angielskie słowo IRREVALANT i poprawiałem to na backendzie w enumach. Aczkolwiek widzę mój błąd w pullu na backendzie bo poszedł main a nie do developa.

Zaraz wrzucę piłka do developa

Reply:
@Alendart ok, masz rację :) na szybko chciałem to poprawić bo BE i FE wtedy się nie zgadzał a łączyłem submity na FE